README.md

```diff
@@ -74,3 +74,15 @@ run this code in external terminal to run tests
  poetry run
 pytest path/to/test_file
 ```
+## Docker
+To build both production and development run
+```
+$ docker build --target development --tag todo-app:dev .
+$ docker build --target production --tag todo-app:prod .
+```
+and these commands to run it
+```
+docker run --env-file ./.env -p 5100:80 --mount type=bind,source="$(pwd)"\todo_app,target=/app/todo_app todo-app:dev
+
+docker run --env-file ./.env -p 5100:80 --mount type=bind,source="$(pwd)"\todo_app,target=/app/todo_app todo-app:prod
```

Comment:
This command just errors for me, with:
```
docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "gunicorn": executable file not found in $PATH: unknown.
```

I think you're missing a few steps to get this working - see my comment on the Dockerfile for more info, but also your command says it's using port 8000 but this command maps port 80